sdk/metricsadvisor/azure-ai-metricsadvisor-perf/README.md

```diff
@@ -0,0 +1,76 @@
+# Azure MetricAdvisor Performance test client library for Java
+
+Represents Performance tests for Azure MetricAdvisor SDK for Java.
+
+## Getting started
+
+### Prerequisites
+
+- Java Development Kit [JDK][jdk_link] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Cognitive Services or Metrics Advisor account][metrics_advisor_account] to use this package.
+
+#### Setup for test resources
+
+You will need the following environment variables for running the tests to access the live resources:
+
+```
+METRICS_ADVISOR_ENDPOINT=<metric advisor service endpoint>
+METRICS_ADVISOR_SUBSCRIPTION_KEY=<metric advisor subscription Key>
+METRICS_ADVISOR_API_KEY=<metric advisor api Key>
+METRICS_ADVISOR_ALERT_CONFIG_ID=<id of a metric advisor alert configuration>
+METRICS_ADVISOR_ALERT_ID=<id of a metric advisor alert>
+METRICS_ADVISOR_DETECTION_CONFIG_ID=<id of a metric advisor detection configurtion>
+METRICS_ADVISOR_INCIDENT_ID=<id of a metric advisor incident>
+```
+
+### Adding the package to your product
+[//]: # ({x-version-update-start;com.azure:azure-ai-metricsadvisor;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-metricsadvisor</artifactId>
+    <version>1.0.0-beta.4</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Key concepts
+
+## Examples
+#### Executing the performance test
+1. Compile the performance project into a standalone jar using the command from the root of the perf project folder
+   ```java
+   mvn clean package
+
+2. Execute the corresponding perf test in the project using the command.
+   ```java
+   java -jar <path-to-packaged-jar-with-dependencies-from-step-1> <options-for-the-test>
```

Comment:
nit: I added an example of what the actual command would look like in FR.

Reply:
example added for all command.